plugin/evm/gossiper.go

```diff
@@ -474,6 +477,8 @@ func (h *GossipHandler) HandleEthTxs(nodeID ids.NodeID, msg message.EthTxsGossip
 			)
 			if err == txpool.ErrAlreadyKnown {
 				h.stats.IncEthTxsGossipReceivedKnown()
+			} else {
+				h.stats.IncEthTxsGossipReceivedError()
```

Comment:
This is a new stat in subnet-evm, but reduces the diff so I figured to take it here.